tpcp/validate/_validate.py

```diff
@@ -187,19 +188,83 @@ def cross_validate(
         )
     assert results is not None  # For the typechecker
     results = _aggregate_final_results(results)
-    score_results = {}
+
     # Fix the formatting of all the score results
-    for name in ["test_scores", "test_single_scores", "train_scores", "train_single_scores"]:
-        if name in results:
-            score = results.pop(name)
-            prefix = ""
-            if "_" in name:
-                prefix = name.rsplit("_", 1)[0] + "_"
-            score = _normalize_score_results(score, prefix)
-            # We use a new dict here, as it is unsafe to append a dict you are iterating over
-            score_results.update(score)
+    score_results = _reformat_scores(
+        ["test_scores", "test_single_scores", "train_scores", "train_single_scores"], results
+    )
+    results.update(score_results)
+
+    return results
+
+
+def validate(
+    pipeline: Pipeline,
+    dataset: Dataset,
+    *,
+    scoring: Optional[Union[Callable, Scorer]] = None,
+    n_jobs: Optional[int] = _Default(None),
+    verbose: int = _Default(0),
+    pre_dispatch: Union[str, int] = _Default("2*n_jobs"),
+    progress_bar: bool = _Default(True),
+):
+    """Evaluate a pipeline on a dataset without any optimization.
+
+    Parameters
+    ----------
+    pipeline
+        A :class:`~tpcp.Pipeline` to evaluate on the given data.
+    dataset
+        A :class:`~tpcp.Dataset` containing all information.
+    scoring
+        A callable that can score a single data point given a pipeline.
+        This function should return either a single score or a dictionary of scores.
+        If scoring is `None` the default `score` method of the optimizable is used instead.
+    n_jobs
+        Number of jobs to run in parallel.
+        One job is created per datapoint.
+        The default (`None`) means 1 job at the time, hence, no parallel computing.
+    verbose
+        The verbosity level (larger number -> higher verbosity).
+        At the moment this only effects `Parallel`.
+    pre_dispatch
+        The number of jobs that should be pre dispatched.
+        For an explanation see the documentation of :class:`~joblib.Parallel`.
+    progress_bar
+        True/False to enable/disable a `tqdm` progress bar.
+    """
+    scoring_args = {"n_jobs": n_jobs, "verbose": verbose, "pre_dispatch": pre_dispatch, "progress_bar": progress_bar}
+    # iterate over args that will be passed to Scorer
+    for arg in scoring_args:
+
+        # when a Scorer instance is provided, the respective arguments were already set
+        if isinstance(scoring, Scorer) and not isinstance(scoring_args[arg], _Default):
+            raise ValueError(  # noqa: TRY004
+                f"Argument `{arg}` was already set in the scorer. You can only set it once."
```

Comment:
Maybe a little bit more information about why this happens:

You passed a explicit Scorer object for the scoring parameter. In this case we expect multiprocessing parameter to be configured directly on the Scorer instance. However, you specified multi-processing related parameters ({list(scoring_args.keys()}) by passing them directly to the validate function. Namely, you passed {arg}={val} to validte. Instead, pass these parameters to your Scorer during initialization or by using `set_params`

(or soemthing similar)